js/utilities.js

```diff
@@ -384,7 +384,14 @@ var ERMrest = (function(module) {
 
             try {
                 var col = columns.get(k);
-                keyValues[k] = col.formatvalue(data[k], { context: context });
+                //If the column type is json or jsonb, then insert the raw data instead of formatted data.
+                //Inserting a formatted data would lead to show blank value instead of "null" value in UI
+                if(col.type.name===('json'||'jsonb') && data[k]===null){
```

Comment:
You don't need this check here. It should go to `formatvalue`, so it can call the [`_getNullvalue`](https://github.com/informatics-isi-edu/ermrestjs/blob/8b1f4d3e5f85d7e72a3127aee0ef523c9cd760a7/js/utilities.js#L318) function. This function is using [display annotation](https://github.com/informatics-isi-edu/ermrest/blob/master/user-doc/annotation.md#2015-display) to find the null value. 

JSON column value will be null in two cases:
1. Actual "null" has been inserted into the JSON column in database.
2. The value doesn't exist.

And since we cannot differentiate between these two cases, it's ok to not show exactly "null" and apply the same logic that we have for showing null value.